contracts/RootChain.sol

```diff
@@ -111,6 +111,19 @@ contract RootChain {
         uint256 _spendingTxSignatures,
         uint256 _spendingTxConfirmationSignatures
     ) public {
+		uint256 eUtxoPos = _encodedSpendingTx.getUtxoPos(_exitingUtxoPosition);
+
+        uint256 _txindex  = (_spendingTxPosition % 1000000000) / 10000;
+        bytes32 root = plasmaChain[_spendingTxPosition / 1000000000].root; //need to make plasma chain/child chain
```

Comment:
I think it'd be cleaner to separate the `blknum` variable here instead of doing it in one line.